Dockerfile

```diff
@@ -27,19 +29,27 @@ RUN go get -u -v sigs.k8s.io/aws-iam-authenticator/cmd/aws-iam-authenticator
 
 ARG DOCKER_VERSION=18.09.7
 RUN curl -L -o- https://download.docker.com/linux/static/stable/x86_64/docker-$DOCKER_VERSION.tgz | tar -zxvf - && cp docker/docker \
-    /usr/bin/docker && chmod 755 /usr/bin/docker
+    /usr/local/bin/docker && chmod 755 /usr/local/bin/docker
 
 ARG KUBE_VERSION=1.15.6
-RUN	curl -L -o /bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v$KUBE_VERSION/bin/linux/amd64/kubectl \
-	&& chmod 755 /bin/kubectl
+RUN curl -L -o /usr/local/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v$KUBE_VERSION/bin/linux/amd64/kubectl \
+    && chmod 755 /usr/local/bin/kubectl
 
 ARG HELM_VERSION=2.16.1
-RUN	curl -L -o- https://storage.googleapis.com/kubernetes-helm/helm-v$HELM_VERSION-linux-amd64.tar.gz | tar -zxvf - && cp linux-amd64/helm \
-	/bin/helm && chmod 755 /bin/helm && helm init --client-only
+RUN curl -L -o- https://storage.googleapis.com/kubernetes-helm/helm-v$HELM_VERSION-linux-amd64.tar.gz | tar -zxvf - && cp linux-amd64/helm \
+    /usr/local/bin/helm && chmod 755 /usr/local/bin/helm && helm init --client-only
+
+ARG HELM3_VERSION=3.3.0
```

Comment:
Add helm3

Reply:
Does anything use it yet?

Reply:
Nope, just allowing us to support the migration, and for people to use things that require it